### PR TITLE
Execute lint step using bundler

### PR DESCRIPTION
**Pull request by djbe**
includes some xcode9 changes

### CODE REVIEW

File: Templates.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -216,7 +216,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ ! $CI ]]; then\n    rake lint:tests\n    rake lint:output\nfi";
+			shellScript = "if [[ ! $CI ]]; then\n  [[ -r ~/.profile ]] && source ~/.profile\n  [[ `which rbenv` ]] && eval \"$(rbenv init - --no-rehash sh)\"\n  if [[ `which bundler` ]]; then\n    bundle exec rake lint:tests\n    bundle exec rake lint:output\n  else\n    echo \"warning: bundler not found, can't lint code. Please install bundler then run bundle install.\"\n  fi\nfi";
```
Comment (AliSoftware) [inline Templates.xcodeproj/project.pbxproj]: I'm not using `rbenv` so not sure why that would be necessary (if it always is, why isn't it in your `~/.profile` for every new shell?), but if we really need to add that here for `rbenv` users, then we should apply that to other repositories too.

Comment (djbe) [reply Templates.xcodeproj/project.pbxproj]: I had to add this for myself, I use `zsh` which doesn't use `.profile`.

Comment (AliSoftware) [reply Templates.xcodeproj/project.pbxproj]: Right. I use zsh too, but I include the `.profile` in my `.zshrc`:

```
[[ -r ~/.profile ]] && source ~/.profile
```

Comment (djbe) [reply Templates.xcodeproj/project.pbxproj]: Right, but I don't use a `.profile` anymore. I have zprezto, which loads `rbenv` automatically. No idea what happens if I try to load `rbenv` twice.

Comment (AliSoftware) [reply Templates.xcodeproj/project.pbxproj]: Why do we need to init `rbenv` in the first place, while we're using system ruby for all our Rakefiles and ruby scripts?

Comment (djbe) [reply Templates.xcodeproj/project.pbxproj]: I get errors with the bundler stuff. I'm telling you, my ruby stuff is always messed up, it's cursed 😢 
I'll add that rbenv in my .profile, hopefully nothing else breaks 🤞

**Pull request merged**